test/views/ChoiceViewSpec.scala

```diff
@@ -90,16 +90,16 @@ class ChoiceViewSpec extends AppViewSpec with ChoiceMessages with CommonMessages
 
       val backButton = createView().getElementById("link-back")
 
-      backButton.text() must be(messages(backCaption))
-      backButton.attr("href") must be(controllers.routes.StartController.displayStartPage().url)
+      backButton.text() mustBe messages(backCaption)
+      backButton.attr("href") mustBe controllers.routes.StartController.displayStartPage().url
```

Comment:
Maybe matcher for linking - more like separate PR, but just for record. I think about something like
```
backButton must linkTo(controllers.routes.StartController.displayStartPage())
```

Reply:
yes, good idea, but another ticket, we can raise sticker on the tech board

Reply:
We already have a Matcher for those. We probably just need to update the code, although this is not part of the ticket

Reply:
This has been addressed.